src/com/gh4a/activities/SearchActivity.java

```diff
@@ -128,6 +137,28 @@ protected boolean onError(Exception e) {
         }
     };
 
+    private LoaderManager.LoaderCallbacks<Cursor> mSuggestionCallback = new LoaderManager.LoaderCallbacks<Cursor>() {
+
+        @Override
+        public Loader<Cursor> onCreateLoader(int id, Bundle args) {
+            return new CursorLoader(getApplicationContext(), SuggestionsProvider.Columns.CONTENT_URI,
+                    new String[]{SuggestionsProvider.Columns._ID, SuggestionsProvider.Columns.SUGGESTION},
+                    SuggestionsProvider.Columns.TYPE + " = ? and " + SuggestionsProvider.Columns.SUGGESTION + " like ?",
+                    new String[]{String.valueOf(mSearchType.getSelectedItemPosition()), mQuery + "%"},
```

Comment:
I'll check this, but as far as I remember onCreateLoader is called from the main that.